### PR TITLE
Support for Super-Res and Reference Frame Sacling in Decoder

### PR DESCRIPTION
**Pull request by nijilk**

### CODE REVIEW

**Comment by nijilk** [comment]
@joelsole @hassount 
Please review

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,8 @@
 - Decoder Post Processing Filters support
 - Decoder Compound Mode (InterInter & InterIntra) Tool support
 - Decoder Film Grain support
+- Decoder SuperRes support
+- Decoder Reference Frame Scaling support
```
Comment (hassount) [inline CHANGELOG.md]: the change log does not seem to be uptodate with master, can you please rebase ?

Comment (hassount) [reply CHANGELOG.md]: @nijilk

Comment (nijilk) [reply CHANGELOG.md]: Sure.
We will fix soon.

**Pull request merged**